### PR TITLE
Enhance: Rework debug view

### PR DESCRIPTION
**Pull request by cjellick**
Signed-off-by: Craig Jellick <craig@acorn.io>

### CODE REVIEW

**Review by ryanhopperlowe — DISMISSED** [review]

**Review by tylerslaton — APPROVED** [review]
Pretty big PR but I couldn't see anything I'd personally change. I like the use of react json tree here, its a natural fit.

**Comment by tylerslaton** [comment]
Side note - when I tested this locally I saw a severity high issue in the npm audit. Will have a follow-up PR that patches that up.

File: components/chat/messages/calls/callFrames.tsx [hunk 1]
```diff
@@ -0,0 +1,310 @@
+import React, { useState, useRef } from 'react';
+import { Button, Tooltip } from '@nextui-org/react';
+import type { CallFrame } from '@gptscript-ai/gptscript';
+import { FaChevronUp, FaChevronDown } from 'react-icons/fa';
+import { AiOutlineLoading3Quarters } from 'react-icons/ai';
+import dynamic from 'next/dynamic';
+import { JSONTree } from 'react-json-tree';
+
+const CallFrames = ({ calls }: { calls: Record<string, CallFrame> | null }) => {
+  if (!calls) return null;
```
[the remark this answers is not in the document]
Comment (cjellick) [reply components/chat/messages/calls/callFrames.tsx:9]: I dont follow. Isn't this a short-circuit, the point of which is to just quit this function early if calls is null/empty?

[the remark this answers is not in the document]
Comment (cjellick) [reply components/chat/messages/calls/callFrames.tsx:10]: spoke offline. i follow now

**Review by tylerslaton — APPROVED** [review]

**Comment by cjellick** [comment]
Ryan had approved, but wanted a TODO comment. I added that. I'm going to merge now just to close the loop here.

**Pull request merged**